dds/DCPS/RTPS/Spdp.cpp

```diff
@@ -2774,6 +2779,26 @@ Spdp::SpdpTransport::send(WriteFlags flags, const ACE_INET_Addr& local_address)
   if (((flags & SEND_RELAY) || outer->config_->rtps_relay_only()) &&
       relay_address != ACE_INET_Addr()) {
     send(relay_address);
+    ++relay_rtps_send_count_;
+    report_relay();
+  }
+}
+
+void
+Spdp::SpdpTransport::report_relay()
+{
+  const DCPS::MonotonicTimePoint now = DCPS::MonotonicTimePoint::now();
+  if (DCPS::DCPS_debug_level >= 4 && now > last_relay_report_ + DCPS::TimeDuration(300)) {
```

Comment:
Every 5 minutes? Could this get called as part of a periodic tasks rather than having every call to send_i potentially need to make these checks?